tst.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "provider": "openai",
```

Comment:
This will be the configuration in the target repo, not the `tst-cli` repo.

Reply:
Okay, I was able to confirm. This was intended for the target repo. However, we will "eat our own dogfood" (https://en.wikipedia.org/wiki/Eating_your_own_dog_food) for creating unit tests in this repo as well.

Let's go ahead and keep this one.